### PR TITLE
Add definitions of rounding mode

### PR DESCRIPTION
**Pull request by 2011eric**

### CODE REVIEW

File: src/decode.c [hunk 1]
```diff
@@ -1030,7 +1030,7 @@ static inline bool op_op_fp(rv_insn_t *ir, const uint32_t insn)
      */
 
     /* decode R-type */
-    uint8_t funct3 = decode_funct3(insn);
+    ir->funct3 = decode_funct3(insn);
```
Comment (jserv) [inline src/decode.c]: Rewrite as following:
```c
uint8_t funct3 = ir->funct3 = decode_funct3(insn);
```

Then, we can avoid accessing `ir->funct3` later.

File: src/softfloat.h [hunk 2]
```diff
@@ -24,7 +24,14 @@ enum {
     FFLAG_UNDERFLOW   = 0b00000000000000000000000000000010,
     FFLAG_INEXACT     = 0b00000000000000000000000000000001,
     //                    ....xxxx....xxxx....xxxx....xxxx
-    RV_NAN            = 0b01111111110000000000000000000000
+    RV_NAN            = 0b01111111110000000000000000000000,
+    //                    ....xxxx....xxxx....xxxx....xxxx
```
Comment (jserv) [inline src/softfloat.h]: I am not sure if the following identifiers make sense. Can you explicitly use them somewhere?

Comment (2011eric) [reply src/softfloat.h]: Maybe I can use switch case to compare them directly when I need to.
Should I delete this?

Comment (jserv) [reply src/softfloat.h]: You can implement the decoding for rounding mode in the proposed changes.

File: README.md [hunk 3]
```diff
@@ -23,6 +23,11 @@ access to all of its features. Your target system must have a functional [SDL2 l
 * macOS: `brew install sdl2`
 * Ubuntu Linux / Debian: `sudo apt install libsdl2-dev`
 
+Install [RISCOF](https://riscof.readthedocs.io/en/stable/installation.html#install-riscof).
```
Comment (jserv) [inline README.md]: You shall submit another pull request for RISCOF.

Comment (jserv) [reply README.md]: In this pull request, let's concentrate on rounding mode.

File: src/softfloat.h (hunk 2 above)
Comment (jserv) [reply src/softfloat.h:28]: Any progress?

**Review by jserv — CHANGES_REQUESTED** [review]
Use `git rebase` rather than `git merge`.
